processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -41,6 +41,10 @@
   @JsonProperty
   private String hadoopContainerDruidClasspath = null;
 
+  //Only applicable when hadoopContainerDruidClasspath is explicitly specified.
+  @JsonProperty
+  private boolean addExtensionsToHadoopContainer = false;
```

Comment:
these configs don't work via the constructor , not sure where to add such check .. we can add it to the getter of addExtensionsToHadoopContainer probably but doesn't seem right.